etc/SuperBuild/CMakeLists.txt

```diff
@@ -93,6 +94,10 @@ endif()
 if(TLRENDER_EXR)
     list(APPEND TLRENDER_EXTERNAL_DEPS OpenEXR)
 endif()
+if(TLRENDER_VPX AND TLRENDER_FFMPEG AND NOT WIN32)
+    list(APPEND TLRENDER_EXTERNAL_DEPS YASM)
```

Comment:
NASM is already a dependency, can it be used instead of YASM?

Reply:
Yes, NASM can be used, but LibVPX recommends YASM for performance.

Reply:
I don't really want to have multiple assemblers as dependencies, can the others also use YASM (jpegturbo, ffmpeg)?

Reply:
Not sure.  I left it with NASM as default and YASM as an optional dependency you can turn on.